chain/cmtx/utils.go

```diff
@@ -39,7 +39,7 @@ func NewTxBuilder(txEncoder sdk.TxEncoder) authtypes.TxBuilder {
 	fee, _ := sdk.ParseCoins("")
 	gasPrices, _ := sdk.ParseDecCoins("")
 	// TODO: Remove hard code gas limit and gas adjustment
-	return authtypes.NewTxBuilder(txEncoder, 0, 0, 10000000, 1, false, "bandchain", "", fee, gasPrices)
+	return authtypes.NewTxBuilder(txEncoder, 0, 0, 20000000, 1, false, "bandchain", "", fee, gasPrices)
```

Comment:
How big can the code be?

Reply:
Current logic.rs (include every lib) use 10342549 from induction It will handle 2 bigger size of code.